pkg/commands/server/testdata/capabilities.success.json.golden

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "Plugin": "vaporio/emulator-plugin",
```

Comment:
i'll make a note below, but for output consistency, we'll want the keys to be lower cased -- that is done via struct annotations